toyml/classification/naive_bayes.py

```
@@ -81,17 +80,25 @@ def _log_likelihood(self, sample: list[FeatureValue]) -> dict[Class, float]:
+        ...     [5.92, 165, 10],
+        ...     [5.00, 100, 6],
+        ...     [5.50, 150, 8],
+        ...     [5.42, 130, 7],
+        ...     [5.75, 150, 9],
+        ... ]
         >>> clf = GaussianNaiveBayes().fit(dataset, label)
         >>> clf.predict([6.00, 130, 8])
         1
 
-    """
+    """  # noqa: E501
```

Ruff (0.8.2)

Unused `noqa` directive (unused: `E501`)

Remove unused `noqa` directive

(RUF100)